<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>load config</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>save config</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveAs">
        <property name="text">
         <string>save as..</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnPreview">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCreateImage">
        <property name="text">
         <string>Create Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>File paths</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Signing private key:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>relative To:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Encryption key file:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Target file:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Hex file:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Version info header:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="rbtn_crypt_plain">
          <property name="text">
           <string>No encryption</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QRadioButton" name="rbtn_crypt_aes">
          <property name="text">
           <string>AES-128 encryption</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="edt_encryptionKey"/>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QLineEdit" name="edt_signPrivateKey"/>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QLineEdit" name="edt_targetFile"/>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="edt_versionInfoHeader"/>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="edt_hexFile"/>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="edtRelative">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Version Info Keywords</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Version</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="edt_gitdate">
          <property name="text">
           <string>GITUNIX</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="edt_name">
          <property name="text">
           <string>VERSION_INFO_NAME</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Git Hash</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edt_githash">
          <property name="text">
           <string>GITHASH</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="edt_version">
          <property name="text">
           <string>VERSION_INFO_VERSION</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Git Date Unixtime</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Image info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="1">
        <widget class="QLabel" name="lblGitDate">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="lblVersion">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="QLabel" name="lblSize">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QLabel" name="lblEntryPoint">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Config file:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="5">
        <widget class="QLabel" name="lblFileName">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Entry point:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Git Date:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Git Hash:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Version:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lblGitHash">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKey">
    <property name="title">
     <string>Key</string>
    </property>
    <addaction name="actionCreate_AES_128_key"/>
    <addaction name="actionCreate_RSA_key_for_signature"/>
   </widget>
   <addaction name="menuKey"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_AES_128_key">
   <property name="text">
    <string>Create AES-128 key for encryption</string>
   </property>
  </action>
  <action name="actionCreate_RSA_key_for_signature">
   <property name="text">
    <string>Create RSA key for signature</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
